source/loop/statsd.rst

```diff
@@ -0,0 +1,36 @@
+| Name                                                  | Type     | Description                                                      |
++=======================================================+==========+==================================================================+
+| ``loop.activated-users``                              | counter  | Count new Hawk Session created for a user.                       |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.call-urls``                                    | counter  | Count new call-url creation.                                     |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.simplepush.call``                              | counter  | Count calls made to a SimplePush URL.                            |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.simplepush.call.(success|failures)``           | counter  | Count SimplePush calls success and failures.                     |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.simplepush.call.{reason}``                     | counter  | Count calls made to a SimplePush URL for a given reason.         |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.simplepush.call.{reason}.(success|failures)``  | counter  | Count calls success and failures on SP urls for a given reason.  |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.aws.write``                                    | timer    | Time AWS S3 write calls.                                         |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.aws.read``                                     | timer    | Time AWS S3 read calls.                                          |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.aws.remove``                                   | timer    | Time AWS S3 object deletion calls.                               |
++-------------------------------------------------------+----------+------------------------------------------------------------------+
+| ``loop.filesystem.write``                             | timer    | Time Filesystem write calls.                                     |
```

Comment:
what file system writes exactly ? if it's always for the same purpose, let's be specific here